From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

swikrutibose committed:
(ForumsTeam) Broken link for "az webapp log tail"

In "Stream diagnostic logs" section, "az webapp log tail" link is broken(404). Please replace with the correct URL: "/cli/azure/webapp/log?view=azure-cli-latest#az_webapp_log_tail".

v-viksur committed:
(ForumsTeam) Requesting to update the hyperlink 

In this documentation "perform these steps for VMs created with the Classic deployment model" is hyperlinked with ARM deployment model. Requesting to update the hyperlink.

v-dotren committed:
Merged Dan's changes w my revisions, ready for tech review

v-dotren committed:
Added TOC entry for batch-docker-container-workloads.md

v-dotren committed:
Incorporated Xing and Matt review feedback

NumberByColors committed:
Rename all instances of Mobile Center to App Center in the Application Insights documentation.

NumberByColors committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into Rename-Mobile-Center

mahesh-unnikrishnan committed:
New article on enabling AAD-DS using PowerShell.

NumberByColors committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into Rename-Mobile-Center

mahesh-unnikrishnan committed:
Update code snippets section.

mahesh-unnikrishnan committed:
TOC integration.

mahesh-unnikrishnan committed:
Full script to create AAD-DS.

mahesh-unnikrishnan committed:
Cleanup script.

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mahesh-unnikrishnan committed:
update post-config/next steps

mahesh-unnikrishnan committed:
acrolinx fixes.

v-dotren committed:
Incorporated Dan's review feedback

Commit:
added not supported to TokenRestricted

Commit:
added note

mmacy committed:
Merge branch 'master' of https://github.com/mmacy/azure-docs-pr

nberdy committed:
Update how-to-control-access.md

Fixing broken link

mmacy committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

mmacy committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

v-dotren committed:
Incorporated Xing's final review feedback

Commit:
Updated owner requirement

Commit:
Contributor access for setup

SnehaGunda committed:
fixing script

Commit:
add Databricks tile for Azure doc hub page

kanshiG committed:
Fix issues with syntax and language and index 

Cassandra requires schema.

nitinme committed:
Moving some pieces around

mimig1 committed:
Cosmos DB: Updates to table-support topic

mimig1 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 11142017tablega

mimig1 committed:
Cosmos DB: Remove mention of Table preview.

mimig1 committed:
Cosmos DB: Remove preview mentions from Spark.

nitinme committed:
minor update

mimig1 committed:
Cosmos DB: Updates to Cassandra .NET QS

kgremban committed:
Updated code for IoT Edge csharp tutorial - to master

kgremban committed:
Updated docker steps - to master

kgremban committed:
Added docker logs - to master

kgremban committed:
Change ASA link - to master

kgremban committed:
Elio's changes to gateway device

kgremban committed:
Ted's fixes to AML tutorial

kanshiG committed:
Updated intro for correctness of index

kgremban committed:
clean up

Rajani-Janaki-Ram committed:
urls & file server

kanshiG committed:
Minor changes.

kanshiG committed:
Update information about certificate

We need to use Microsoft certificate rather than create our own.

Rajani-Janaki-Ram committed:
urls & file server

kanshiG committed:
Updated information about certificate

We need to use certificate from https://docs.microsoft.com/en-us/azure/java-add-certificate-ca-store

mimig1 committed:
Cosmos DB: MongoDB feature support article.

mimig1 committed:
added link from mongodb-introduction

Rajani-Janaki-Ram committed:
urls & file server

mimig1 committed:
Acrolinx fixes

Rajani-Janaki-Ram committed:
urls & file server

mimig1 committed:
Cosmos DB: Updating includes for portal changes

mimig1 committed:
update

mahesh-unnikrishnan committed:
Update active-directory-ds-enable-using-powershell.md

mahesh-unnikrishnan committed:
Update active-directory-ds-enable-kcd.md

mahesh-unnikrishnan committed:
Update active-directory-ds-getting-started-password-sync-synced-tenant.md

mahesh-unnikrishnan committed:
Update active-directory-ds-troubleshooting.md

mahesh-unnikrishnan committed:
Update active-directory-ds-enable-kcd.md

mahesh-unnikrishnan committed:
Update active-directory-ds-troubleshooting.md

pareshmu committed:
Updating how to NPM for ER

Updating the whitelisting URL & adding a note

Commit:
Update tutorial-csharp-module.md

Commit:
Update tutorial-deploy-function.md

tynevi committed:
Merge pull request #27882 from shizn/patch-4

Update tutorial-deploy-function.md

tynevi committed:
Merge pull request #27881 from shizn/patch-3

Update tutorial-csharp-module.md

tynevi committed:
Merge pull request #27879 from pareshmu/patch-2

Updating how to NPM for ER

tynevi committed:
Merge pull request #27876 from mahesh-unnikrishnan/patch-18

Update active-directory-ds-troubleshooting.md

tynevi committed:
Merge pull request #27875 from mahesh-unnikrishnan/patch-17

Update active-directory-ds-getting-started-password-sync-synced-tenan…

tynevi committed:
Merge pull request #27874 from mahesh-unnikrishnan/patch-15

Update active-directory-ds-enable-kcd.md

tynevi committed:
Merge pull request #27858 from kgremban/edge-amlfix

Ted's fixes to AML tutorial

tynevi committed:
Merge pull request #27855 from kgremban/elio-master

Elio's changes to gateway device

tynevi committed:
Merge pull request #27854 from kgremban/edge-ASA-master

Change ASA link - to master

tynevi committed:
Merge pull request #27853 from kgremban/edge-viewdata-master

Added docker logs - to master

tynevi committed:
Merge branch 'master' into edge-ui2-master

tynevi committed:
Merge pull request #27851 from kgremban/edge-certs-master

Updated code for IoT Edge csharp tutorial - to master

tynevi committed:
Merge pull request #27850 from mimig1/11142017cassnetqs1

Cosmos DB: Updates to Cassandra .NET QS

tynevi committed:
Merge pull request #27849 from mimig1/11142017sparkga

Cosmos DB: Remove preview mentions from Spark.

tynevi committed:
Merge pull request #27848 from mimig1/11142017tableps

Cosmos DB: Remove mention of Table preview.

tynevi committed:
Merge pull request #27846 from nitinme/nitinme-databricks-updates

Moving some pieces around

tynevi committed:
Merge pull request #27839 from SnehaGunda/policy

fixing script

tynevi committed:
Merge pull request #27838 from raymondlaghaeian/patch-16

Contributor access for setup

tynevi committed:
Merge pull request #27835 from raymondlaghaeian/patch-15

Updated owner requirement

tynevi committed:
Merge pull request #27813 from nberdy/patch-1

Update how-to-control-access.md

tynevi committed:
Merge pull request #27808 from johndeu/master

added not supported to TokenRestricted

tynevi committed:
Merge pull request #26469 from v-viksur/patch-6

(ForumsTeam) Requesting to update the hyperlink

tynevi committed:
Merge pull request #24896 from swikrutibose/patch-1

(ForumsTeam) Broken link for "az webapp log tail"

tynevi committed:
Merge pull request #27852 from kgremban/edge-ui2-master

Updated docker steps - to master

mimig1 committed:
Cosmos DB: Corrected link.

mimig1 committed:
Cosmos DB: Corrected nuget history

tynevi committed:
Merge pull request #27840 from v-savila/upd1126019

add Databricks tile for Azure doc hub page

spelluru committed:
updating files for usability study

mawah committed:
Added file share creation command

v-dotren committed:
Fixed blocking style issues

zanasrin committed:
Update create-cassandra-python.md

1. Changed to LCA approved names
2. Fixed indentation
3. Changed to network topology in create keyspace
4. Added packages to allow ssl validation against the default store

mmacy committed:
Merge branch 'master' of https://github.com/mmacy/azure-docs-pr

zanasrin committed:
Update create-cassandra-nodejs.md

1. Changed to LCA approved names
2. Changed to Network Topology

dominicbetts committed:
Test

mmacy committed:
[aks] --agent-count > --node-count

zanasrin committed:
Update create-cassandra-python.md

zanasrin committed:
Update create-cassandra-python.md

@mimig1

CarlRabeler committed:
Update sql-database-design-first-database.md

zanasrin committed:
Update create-cassandra-python.md

MGoedtel committed:
updated intro paragraph per PM feedback

CarlRabeler committed:
Update sql-database-design-first-database-csharp.md

zanasrin committed:
Update create-cassandra-python.md

rjagiewich committed:
Merge pull request #27859 from Rajani-Janaki-Ram/master

urls & file server

zanasrin committed:
Update create-cassandra-nodejs.md

tynevi committed:
Merge pull request #27899 from MGoedtel/UpdateRunbookExec1115

updated intro paragraph per PM feedback

tynevi committed:
Merge pull request #27895 from dominicbetts/patch-1

Minor fix

tynevi committed:
Merge pull request #27891 from mawah/patch-5

Added file share creation command

tynevi committed:
Merge pull request #27888 from mimig1/patch-165

Cosmos DB: Corrected link.

tynevi committed:
Merge pull request #27863 from kanshiG/patch-16

Updated information about certificate

tynevi committed:
Merge pull request #27862 from kanshiG/patch-15

Update information about certificate

tynevi committed:
Merge pull request #27860 from kanshiG/patch-14

Minor changes.

rjagiewich committed:
Merge pull request #27865 from mimig1/11142017mongocompat

Cosmos DB: MongoDB feature support article

ktoliver committed:
Merge pull request #27868 from mimig1/11142017acctnames

Cosmos DB: Updating includes for portal changes

zanasrin committed:
Update create-cassandra-python.md

mmacy committed:
[aks] more 'agent' > 'node'

ktoliver committed:
Merge pull request #27897 from CarlRabeler/patch-362

Update sql-database-design-first-database.md

zanasrin committed:
Merge branch 'master' into patch-2

GitHubber17 committed:
Merge pull request #27844 from kanshiG/patch-13

Fix issues with syntax and language and index

ktoliver committed:
Merge pull request #27847 from mimig1/11142017tablega

Cosmos DB: Updates to table-support topic

ktoliver committed:
Merge pull request #27896 from mmacy/aks-node-count-param

[aks] agent > node

rjagiewich committed:
Merge pull request #27667 from mahesh-unnikrishnan/master

New article on enabling AAD-DS using PowerShell.

ganesr committed:
Update expressroute-gateway-classic-ps-include.md

rjagiewich committed:
Merge pull request #27890 from spelluru/adfps1115

updating files for usability study

ktoliver committed:
Merge pull request #27894 from zanasrin/patch-2

Update create-cassandra-nodejs.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

GitHubber17 committed:
Merge pull request #26879 from v-dotren/batch-containers-tr

Batch containers article tech review

GitHubber17 committed:
Merge pull request #27906 from ganesr/patch-3

Update expressroute-gateway-classic-ps-include.md

GitHubber17 committed:
Merge pull request #27898 from CarlRabeler/patch-363

Update sql-database-design-first-database-csharp.md

steveesp committed:
Azure kernel GA v2

GitHubber17 committed:
Merge pull request #27893 from zanasrin/patch-1

Update create-cassandra-python.md

steveesp committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vnet-optimal

fixing details about the Azure kernel

rjagiewich committed:
Merge pull request #27653 from NumberByColors/Rename-Mobile-Center

Rename Mobile Center to App Center in the App Insights docs

v-alje committed:
Merge pull request #27910 from MicrosoftDocs/FromPublicRepo

From public repo

v-shils committed:
Merge pull request #27911 from steveesp/vnet-optimal

Vnet optimal
